### DISCUSSION TITLE
Filament Admin Login Error 404 on server

### DISCUSSION THREAD

**Discussion by rajvansh-369** [filamentphp/filament#5341] [Help]
I created Laravel Project and add filament to it 
it is working fine locally at http://127.0.0.1:8000/

I push it to GitHub and clone it to the server 
and run composer update, and also set env('ASSET_URL') in config/livewire.php (livewire.js load sucessfully)
The login panel is opening successfully but while clicking on the login, I got an error message 

index.js:40          POST https://uxdesigns.in/filament/public/livewire/message/filament.core.auth.login 404

![image](https://user-images.githubusercontent.com/106300079/209511821-93a259ad-0f74-4252-b7d1-bfc4f699eb78.png)

![image](https://user-images.githubusercontent.com/106300079/209511851-d492d2e2-d97f-47ec-8009-f10f09968786.png)

I tried everything but still didn't get any solution.
Please suggest to me the solution. 
Thank you

**Comment by ArtMin96** [comment]
Are you using nginx? If so, this will solve the problem.

```
location ^~ /filament {
    try_files $uri $uri/ /index.php?$query_string;
}
```

**Reply by rajvansh-369** [reply]
Thank you, but I'm using Apache 2.4.54

![image](https://user-images.githubusercontent.com/106300079/209533085-79682051-7940-4920-b31f-4d5be93bf5fc.png)

**Reply by ArtMin96** [reply]
Is the login component registered in the service provider?
```php
Livewire::component(Login::getName(), Login::class);
```

**Reply by rajvansh-369** [reply]
No, it is not defined in the service provider. 
Should I define that?
I never defined this before, 

here is my repo 
https://github.com/steve-369/Expenses-Management___.git
It was working a couple of days ago on the server, but after pushing composer.json, I'm getting the above error.
I thought I'm getting this issue after including breezy but when I try to create a new project I'm getting the same error

**Reply by ArtMin96** [reply]
For breezy you don't need to register component.
You can ask here: https://discord.com/channels/883083792112300104/935091455280709672

**Reply by rajvansh-369** [reply]
Ok thank you

**Comment by ramiropiro** [comment]
HI steve, how you solved this, having the same problem with breeze when I try to login on a production server
